lms/djangoapps/discussion_api/permissions.py

```diff
@@ -56,31 +56,36 @@ def get_editable_fields(cc_content, context):
     """
     Return the set of fields that the requester can edit on the given content
     """
-    # Shared fields
-    ret = {"abuse_flagged", "voted"}
-    if _is_author_or_privileged(cc_content, context):
-        ret |= {"raw_body"}
-
-    # Thread fields
-    if cc_content["type"] == "thread":
-        ret |= {"following"}
+    ret = {"abuse_flagged"}
+    if (cc_content["type"] == "thread" and cc_content["closed"]) or \
```

Comment:
Please add a comment here that no edits, except `abuse_flagged` are allowed on closed threads.